apps/api/src/app/widgets/usecases/initialize-session/initialize-session.usecase.ts

```diff
@@ -56,6 +62,7 @@ export class InitializeSession {
         lastName: subscriber.lastName,
         phone: subscriber.phone,
       },
+      feeds,
```

Comment:
Will all the object will be returned here?

class FeedEntity {
  _id?: string;
  name: string;
  _environmentId: string;
  _organizationId: string;
}

If so maybe we can remove all the unneeded parameters and return only feeds? what do you think?